File: oop/src/mvp/view/messages/Messages.java
package mvp.view.messages;



public class Messages implements InterfaceForMessages  {
     private String message;


    public Messages() {
        this.message ="";
    }

    @Override
    public void showMessage(String message) {
        this.message=message;
        toString();
    }






    @Override
    public String toString() {
        System.out.println("сообщение:"+this.message);
        return null;
    }
}

File: oop_ds/src/refactor_by_oop/mvp/view/messages/Messages.kt
package refactor_by_oop.mvp.view.messages

class Messages:IMessages {
    private var message = ""



    override fun showMessage(message: String?) {
        if (message != null) {
            this.message = message
        }
        toString()
    }


    override fun toString(): String {
        println("сообщение:" + this.message)
        return ""
    }
}